examples/logistic.jl

```diff
@@ -19,20 +20,22 @@ y = draw(y_prob)
 
 # Build the model
 sess = Session(Graph())
-X = placeholder(Float64, shape=[nothing, 10])
-Y_obs = placeholder(Float64, shape=[100])
+@tf begin
```

Comment:
The example does need to change, as it uses inconstant sizes; and is a test, which would otherwise fail.

But I can remove the `@tf` because I only added that to help debug, which line had the inconstant sizing.